framework/include/base/MooseError.h

```diff
@@ -99,6 +99,10 @@ class MooseVariableFEBase;
 
 namespace moose
 {
+
+extern bool show_trace;
+extern bool show_multiple;
```

Comment:
I think that all of our globals variables are defined in `Moose.h` right now. If that's the case, I'd think I'd prefer to maintain that organization moving forward instead of having them introduced here. Also, can we add doc strings to these? Since they are global, it would be nice to know what they or for/why they are here and perhaps even information about when/why they get set. It'll help with maintenance in the future.

Reply:
Yeah - these could use some good doc comments.